### ISSUE TITLE
Bug report: Can't open demo with firefox

### ISSUE THREAD

**Issue by kserg9707** [gchq/CyberChef#361]
Bug report:

### Summary
Can't open [demo](https://gchq.github.io/CyberChef/) with firefox on linux and windows. Chrome works fine.

### Example
```
SecurityError: The operation is insecure.
Filename: https://gchq.github.io/CyberChef/main.js
Line: 106
Error: SecurityError: The operation is insecure.
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:60.0) Gecko/20100101 Firefox/60.0
CyberChef version: 8.7.0
```

**Comment by d98762625** [comment]
@kserg9707 are you using Mozilla Firefox 35+ ?

**Comment by kserg9707** [comment]
> @kserg9707 are you using Mozilla Firefox 35+ ?

Yes. I've tried with Firefox versions 60, 61 and 62, every time got same result.

**Comment by d98762625** [comment]
I cannot replicate this on my machine. Can you see if this security feature in Firefox is the culprit?

https://stackoverflow.com/questions/11768221/firefox-websocket-security-issue/12042843#12042843

**Comment by kserg9707** [comment]
> Can you see if this security feature in Firefox is the culprit?

I have enabled this feature, nothing has changed. Getting same error.

**Comment by Edward70891** [comment]
I'm having the same problem on linux with firefox 62 with both hosted and local versions
```
SecurityError: The operation is insecure.
Filename: file:///home/edward/bin/cyberchef.htm
Line: 13925
Error: SecurityError: The operation is insecure.
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:62.0) Gecko/20100101 Firefox/62.0
CyberChef version: 8.7.0s
```

**Comment by n1474335** [comment]
Closing this issue as I believe it was fixed in #403.
